### PR TITLE
Added Swagger2 Documentation Implementation

### PR DESCRIPTION
**Pull request by Chris-Michael-Allen**
Testing the pipeline

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,3 +1,9 @@
 #TODO: Spring profile local
 spring.application.name=ask
-server.port=1337
\ No newline at end of file
+server.port=1337
+
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
```
Comment (AdamDKing) [inline src/main/resources/application.properties]: Is this supposed to be "password"?  Try it

**Review by AdamDKing — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (Chris-Michael-Allen) [reply src/main/resources/application.properties]: No current difference when changing the password

**Review by AdamDKing — CHANGES_REQUESTED** [review]
Please follow Google Style -- indent 2 spaces for a tab rather than 4 and limit lines to 100 characters

**Review by AdamDKing — APPROVED** [review]
Looks good for swagger so far.  Thanks for making the style changes.

**Pull request merged**